src/components/shared/Subscribe.vue

```diff
@@ -86,8 +90,8 @@ export default {
     </div>
     <span>olarak almak için</span>
     <input type="text" v-model="name" placeholder="İsim soyisim" class="username">
-    <input type="text" v-model="email" placeholder="Email">
-    <button class="button" @click="handleSubscribe">
+    <input @keyup.enter.once="handleEnter" type="text" v-model="email" placeholder="Email">
+    <button class="button" @click.once="handleSubscribe">
```

Comment:
You can't use `once` because if there is something wrong with the user-provided data, they want to retry but then the click will not work.

I already see that you added `this.isClosed` logic to `handleSubscribe`. That's good. Now you should remove these `once`s and allow `keyup.enter` and `click` to trigger `handleSubscribe` multiple times and you should also add

```js
if (this.isClosed) {
  return false;
}```

to the very beginning of the `handleSubscribe` to do an early return.

Reply:
I removed `once` and added the lines you suggested (changed `return false` to `return` to fix vue linter error)